code/auxiliaries/DefaultPopulationSettings.m

```diff
@@ -17,30 +17,37 @@
         return
     end
     
-    populationSettings = PKSim.Core.Batch.PopulationSettings;
     gestationalAgeRange = PKSim.Core.CoreConstants.PretermRange;
 
     populationSettings.Species             ='Human';
     populationSettings.Population          ='European_ICRP_2002';
+    
     populationSettings.MinAge              = NaN;
     populationSettings.MaxAge              = NaN;
+    populationSettings.AgeUnit             = 'year(s)';
+    
```

Comment:
Extended with units for age, weight, height etc.
Per default all units are set to internal units for the corresponding dimensions